gpkit/tools/tools.py

```diff
@@ -141,7 +141,7 @@ def composite_objective(*objectives, **kwargs):
         descr = dict(w.descr)
         del descr["value"]
         descr["name"] = "v_{CO}"
-        w_s.append(Variable(value=('sweep', lambda x: 1-x), args=[w], **descr))
+        w_s.append(Variable(value=lambda const: 1-const[w.key], **descr))  # pylint: disable=cell-var-from-loop
```

Comment:
I think pylint might be right here -- are you sure the values of `w` that end up in all these lambdas aren't the same after loop closure?

```python
In [12]: funs = [lambda x: x**i for i in range(4)]

In [13]: [f(2) for f in funs]
Out[13]: [8, 8, 8, 8]
```